SprayAndPrayWeb/Views/Home/Privacy.cshtml

```diff
@@ -110,4 +110,52 @@
     </li>
 </ol>
 
+<div class="1. what information do we collect?">
```

Comment:
class is for bringing in css. You don't want a label inside class.
An example would be like:
<div class="sap-row">
1. What information do.....
<\div>

Where "sap-row" is defined in css file 

For now you can remove class altogether

Reply:
I fixed this